pkg/controller/controller_broker.go

```diff
@@ -224,15 +229,35 @@ func (c *controller) reconcileServiceBroker(broker *v1alpha1.ServiceBroker) erro
 		}
 		glog.V(5).Infof("Successfully converted catalog payload from ServiceBroker %v to service-catalog API", broker.Name)
 
-		if len(catalog) == 0 {
+		if len(serviceClasses) == 0 {
```

Comment:
where do we check to make sure there's at least one plan per serviceClass?

Reply:
I thought it was done somewhere else, but I'm not seeing it.

Reply:
yea, it'd be gone from validation of each class.

Reply:
we need to add that back in then

Reply:
https://github.com/kubernetes-incubator/service-catalog/pull/1106/files#diff-3c862eb54a8e0e161b534e0c67e5379eR500